<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseStation</class>
 <widget class="QWidget" name="BaseStation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/models/Icons/GPS Receiving-96.png</normaloff>:/models/Icons/GPS Receiving-96.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Ublox</string>
      </attribute>
      <attribute name="toolTip">
       <string>Ublox</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Connect Ublox</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="ubxSerialRefreshButton">
            <property name="toolTip">
             <string>Refresh</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/models/Icons/Refresh-96.png</normaloff>:/models/Icons/Refresh-96.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ubxSerialPortBox"/>
          </item>
          <item>
           <widget class="QSpinBox" name="ubxSerialBaudBox">
            <property name="suffix">
             <string/>
            </property>
            <property name="prefix">
             <string>Baud: </string>
            </property>
            <property name="maximum">
             <number>4000000</number>
            </property>
            <property name="value">
             <number>115200</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>17</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="ubxSerialConnectedLabel">
            <property name="text">
             <string>Not Connected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ubxSerialDisconnectButton">
            <property name="toolTip">
             <string>Disconnect</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/models/Icons/Disconnected-96.png</normaloff>:/models/Icons/Disconnected-96.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ubxSerialConnectButton">
            <property name="toolTip">
             <string>Connect</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/models/Icons/Connected-96.png</normaloff>:/models/Icons/Connected-96.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>ubxSerialPortBox</zorder>
         <zorder>ubxSerialDisconnectButton</zorder>
         <zorder>ubxSerialConnectButton</zorder>
         <zorder>ubxSerialBaudBox</zorder>
         <zorder>ubxSerialRefreshButton</zorder>
         <zorder>ubxSerialConnectedLabel</zorder>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="basePosBox">
         <property name="title">
          <string>Base Station Position</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QCheckBox" name="sendBaseBox">
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>33</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="refGetButton">
              <property name="toolTip">
               <string>Get average from nmea tab</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/models/Icons/Radio Tower-96.png</normaloff>:/models/Icons/Radio Tower-96.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QDoubleSpinBox" name="refSendLatBox">
              <property name="prefix">
               <string>Lat: </string>
              </property>
              <property name="decimals">
               <number>8</number>
              </property>
              <property name="minimum">
               <double>-90.000000000000000</double>
              </property>
              <property name="maximum">
               <double>90.000000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="refSendLonBox">
              <property name="prefix">
               <string>Lon: </string>
              </property>
              <property name="decimals">
               <number>8</number>
              </property>
              <property name="minimum">
               <double>-180.000000000000000</double>
              </property>
              <property name="maximum">
               <double>180.000000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="refSendHBox">
              <property name="prefix">
               <string>h: </string>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>-5000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="refSendAntHBox">
              <property name="prefix">
               <string>Ant h: </string>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="maximum">
               <double>180.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="sendVehiclesBox">
           <property name="text">
            <string>Send to Vehicles</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="tcpServerBox">
           <property name="text">
            <string>TCP Server</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="tcpServerPortBox">
           <property name="prefix">
            <string>Port: </string>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>8200</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>244</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/models/Icons/NMEA-96.png</normaloff>:/models/Icons/NMEA-96.png</iconset>
      </attribute>
      <attribute name="title">
       <string>NMEA</string>
      </attribute>
      <attribute name="toolTip">
       <string>NMEA Client</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Connect NMEA Server</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nmeaServerEdit">
           <property name="text">
            <string>127.0.0.1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="nmeaPortBox">
           <property name="prefix">
            <string>Port: </string>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>2948</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nmeaConnectButton">
           <property name="toolTip">
            <string>Connect</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/models/Icons/Connected-96.png</normaloff>:/models/Icons/Connected-96.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="nmeaSampleLabel">
           <property name="text">
            <string>Samples...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nmeaSampleClearButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/models/Icons/Delete2-96.png</normaloff>:/models/Icons/Delete2-96.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="nmeaBrowser">
         <property name="font">
          <font>
           <family>Monospace</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
